00_SETUP.md

```diff
@@ -68,24 +68,24 @@ And finally, on any platform, to install and activate the navo-workshop
 environment, type:
```

Comment:
One more occurrence of `navo-workshop` on the preceding line. Maybe it could say "to install and activate the conda environment for the workshop, type".

Reply:
Thanks for catching that.  Change made as suggested.